src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double base;
+    private double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return base * height / 2;
```

Comment:
Consider using a constant for the magic number '2' in the area calculation to give it a meaningful name, enhancing code readability and maintainability.